client/components/LearnerViewDirective.js

```diff
@@ -773,7 +801,15 @@ tie.directive('learnerView', [function() {
             for (var i = 0; i < feedbackParagraphs.length; i++) {
               if (feedbackParagraphs[i].isSyntaxErrorParagraph()) {
                 hasSyntaxError = true;
+                clearHighlight();
```

Comment:
Retracting this comment after discussion with @eledavi. There's still a feedback message.